tools/myst/host/package.c

```diff
@@ -84,6 +86,40 @@ static int _add_image_to_elf_section(
     return ret;
 }
 
+static void _make_cpio_archive_from_directory(
+    const char* arg0,
+    const char* dirname,
+    const char* archive)
+{
+    const char* args[] = {arg0, "mkcpio", dirname, archive};
+
+    if (_mkcpio(sizeof(args) / sizeof(args[0]), args) != 0)
+    {
+        _err(
+            "Failed to create root filesystem \"%s\" from directory \"%s\"\n",
+            archive,
+            dirname);
+    }
+}
+
+static void _generate_dummy_cpio_rootfs(const char* archive)
```

Comment:
Please comment on what's the purpose of a dummy cpio archive.

Reply:
I removed this function bud added a comment to the original code that did the same. Perhaps we should remove this hack later.